PodcastGenerator/admin/episodes_ftp_feature.php

```diff
@@ -107,10 +109,14 @@
         }
         // Write XML file
         file_put_contents('../' . $config['upload_dir'] . pathinfo($fname, PATHINFO_FILENAME) . '.xml', $episodefeed);
-        // Regenarte RSS feed
+        $num_added += 1;
+    }
+    if ($num_added > 0) {
```

Comment:
Remove the `0`. Any other value than null is evaluted as `TRUE` by PHP
Therefore
```
if ($num_added) {
```

Reply:
Done.